### PR TITLE
Foliage

### PR DESCRIPTION
**Pull request by abdominator**
added foliage and stalactites. Visual things only.

### CODE REVIEW

File: Game/Resources/lowpoly_bush.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+				"metallic" : "",
+				"normal" : "Resources/Foliage/fern_normal",
+				"roughness" : ""
+			},
+			"Mesh" : 
+			{
+				"model" : "Resources/lowpoly_bush"
+			},
+			"children" : null,
+			"name" : "Entity #1",
```
Comment (marredal) [inline Game/Resources/lowpoly_bush.json]: Give this a name.

File: Game/Resources/lowpoly_bush_scene.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+	"children" : 
+	[
+		{
+			"Material" : 
+			{
+				"albedo" : "Resources/Foliage/fern_albedo",
+				"metallic" : "Resources/Foliage/fern_metallic",
+				"normal" : "Resources/Foliage/fern_normal",
+				"roughness" : ""
```
Comment (marredal) [inline Game/Resources/lowpoly_bush_scene.json]: No texture

**Comment by Ittis** [comment]
iamheman

**Comment by marredal** [comment]
iamheman

File: Game/Resources/lowpoly_bush.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
+{
+	"children" : 
+	[
+		{
+			"DirectionalLight" :
```
Comment (andens) [inline Game/Resources/lowpoly_bush.json]: Won't this insert alot of directional lights into the game when this scene is instantiated?

Comment (abdominator) [reply Game/Resources/lowpoly_bush.json]: Correct. Forgot to clean this up.

**Comment by abdominator** [comment]
iamheman

**Comment by abdominator** [comment]
iamheman

**Comment by abdominator** [comment]
iamheman

**Pull request merged**